src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <title>Register</title>
+ <link rel="stylesheet" href="/css/main.css">
+ <style>
+   label {
+     display: inline-block;
+     width: 100px;
+   }
+ </style>
```

Comment:
Same with the other jsp file's style - should we leave this in the jsp file or separate it out to a CSS file?

Reply:
Disregard this, saw the comments above!